plugin/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "url_plugin"
+version = "0.1.0"
+authors = ["Corey Farwell <coreyf@rwell.org>"]
+
+[lib]
+name = "url_plugin"
+plugin = true
+
+[dependencies.url]
+path = "../"
```

Comment:
I think for publishing to work you need to have a version dep here too. Not sure